configs/baremetal.h

```diff
@@ -137,6 +137,8 @@
 
 #define MBEDTLS_OID_C
 #define MBEDTLS_PLATFORM_C
+#define MBEDTLS_CRC_C
+#define MBEDTLS_VALIDATE_AES_KEYS
```

Comment:
@AndrzejKurek , can we define by yourself crc flag if VALIDATE flag is defined. Like
#if defined(MBEDTLS_VALIDATE_AES_KEYS)
#define MBEDTLS_CRC_C
#end if
I would like to make it easier for users ...

Reply:
`VALIDATE_AES_KEYS` depends on `MBEDTLS_CRC_C`, and thanks to a check in `check_config.h`, if someone doesn't enable `MBEDTLS_CRC_C`, he/she will get a clear message during compilation. I haven't seen this way of enabling dependencies by features that use them in the library anywhere, and I would suggest that we shouldn't do that. If someone disables `MBEDTLS_CRC_C` in the config, it would be a surprise to them that it is defined anyway - it's unexpected.